app/src/main/java/com/gacode/relaunchx/ReLaunch.java

```diff
@@ -1585,37 +1595,13 @@ public boolean onSingleTapConfirmed(MotionEvent e) {
 
 			@Override
 			public boolean onDoubleTap(MotionEvent e) {
-				if (prefs.getBoolean("disableScrollJump", true) == false) {
-					int first = gv.getFirstVisiblePosition();
-					int total = itemsArray.size();
-					first -= (total * app.scrollStep) / 100;
-					if (first < 0)
-						first = 0;
-					gv.setSelection(first);
```

Comment:
Why do you want to remove this functionality?

Reply:
For all I know this functionallity allows you to scroll page up/down by determined % of files (directories), which I doubt anybody use it (but maybe I'm wrong).

Reply:
Yes, I'm thinking of removing it, maybe just removing the settings in next update to see if anyone complains ;-)